deployments/helm/nsm/values.yaml

```diff
@@ -25,4 +25,4 @@ spire:
 
 global:
   # set to true to enable Jaeger tracing for NSM components
-  JaegerTracing: false
+  JaegerTracing: true
```

Comment:
Hmm I would suggest this to be false by default and for development purposes to enable it in the `make helm-install-nsm` target.

Reply:
Good idea. Fixed.